lib/modules/manager/npm/index.ts

```diff
@@ -17,7 +17,6 @@ export const supportsLockFileMaintenance = true;
 
 export const defaultConfig = {
   fileMatch: ['(^|/)package\\.json$'],
-  rollbackPrs: true,
```

Comment:
You'll need to change the docs for this change. 😉 Quote from the `rollbackPrs` config option docs:

> Renovate does not create these rollback PRs by default, with one exception: npm packages get a rollback PR if needed.